javascripts/tagmanager.js

```diff
@@ -754,6 +760,11 @@
                     return '';
                 },
                 getElementAttribute: function (node, attributeName) {
+                    // If the attribute belongs to a masked element, return a masked string.
+                    if (node && TagManager.dom.shouldElementBeMasked(node)) {
```

Comment:
Why have we added masking here ?

Reply:
@AltamashShaikh I added that since there might be Variables that expose data-* attributes that contain sensitive information in Dimensions.

Like this:
![image](https://user-images.githubusercontent.com/11394099/180369553-d15ecc1c-a08d-44b6-80cd-ca41b3ba0de8.png)

With the code in the getElementAttribute function:
![image](https://user-images.githubusercontent.com/11394099/180370090-0c76a6f1-fc38-4c43-b4a1-357ba6cf1cd2.png)

Reply:
Yes, we would mask some common attributes as it was implemented like `title` or `alt` 👍